src/pydap/net.py

```diff
@@ -20,3 +23,46 @@ def raise_for_status(response):
             headers=response.headers,
             comment=response.body
         )
+
+
+def follow_redirect(url, application=None, session=None,
```

Comment:
I think ``get_dict()`` is safe here. My extensive (yet limited) experience with ``requests`` is that things that you expect either work or fail in very transparent ways.

I've played with the cookie setting procedure of ``webob`` and it is a pain. I'm not sure anymore but I think I tested alternatives and this one was the one that worked...